ufc/tests/test-case-boolean-one-of-matches.json

```diff
@@ -11,9 +11,7 @@
       "assignment": 1,
       "assignmentDetails": {
         "value": 1,
-        "environment": {
-          "name": "Test"
-        },
+        "environmentName": "Test",
```

Comment:
keeping this as a top-level field so that it isn't nested (when logging to Chrome console for instance).

Reply:
Oh true, nested objects are a hassle to sift through in console logs 👍